### PR TITLE
UIOR-182 Item Status upon receipt should be In process

### PR DESCRIPTION
**Pull request by aliaksei-chumakou**
## Purpose
Item Status upon receipt/checkin: When piece is received item status should transition from "On order" to "In process".
https://issues.folio.org/browse/UIOR-182

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: translations/ui-orders/en.json [hunk 1]
```diff
@@ -501,6 +501,8 @@
   "receiving.cancelBtn": "Cancel",
   "receiving.dateOrdered": "Date Ordered",
   "receiving.format": "Format",
+  "receiving.itemStatus.inProcess": "In process",
+  "receiving.itemStatus.onOrder": "On order",
   "receiving.itemStatus.received": "Received",
```
Comment (craigmcnally) [inline translations/ui-orders/en.json]: I think "Received" can be removed as an itemStatus.  It has been replaced by "In process"

File: src/components/Receiving/const.js [hunk 2]
```diff
@@ -2,4 +2,5 @@ export const PIECE_STATUS_EXPECTED = 'Expected';
 export const PIECE_STATUS_RECEIVED = 'Received';
 export const ITEM_STATUS = {
```
Comment (craigmcnally) [inline src/components/Receiving/const.js]: Just curious... why define separate variables for the piece statuses, but an object for the item statuses?

File: translations/ui-orders/en.json (hunk 1 above)
Comment (aliaksei-chumakou) [reply translations/ui-orders/en.json]: ok

File: src/components/Receiving/const.js (hunk 2 above)
Comment (aliaksei-chumakou) [reply src/components/Receiving/const.js]: historically we had no info about available statuses, so one constant was fine. But importing one object with all statuses is more handy, so we might refactor piece statuses at some point in the future

**Pull request merged**